openapi/2023-12.yaml

```diff
@@ -1400,3 +1473,74 @@ components:
                 - optin
               description: Type of source
               example: optin
+    PhoneNumbersToListRequest:
+      title: PhoneNumberToListRequest
+      x-stoplight:
+        id: gwpfudrp4wj2w
+      type: object
+      description: Handling phoneNumbers add/remove to a specified list
+      properties:
+        data:
+          type: array
+          x-stoplight:
+            id: 5m8e1f3mvnnlc
+          items:
+            x-stoplight:
+              id: bviei5et1bcu1
+            type: object
+            properties:
+              type:
+                x-stoplight:
```

Comment:
can we skip generating these x-stoplight fields somehow? it makes reviewing the code harder

Reply:
https://docs.stoplight.io/docs/platform/hosted-docs/stoplight-urls#stable-id-guidelines